src/pages/Reviews.jsx

```diff
@@ -12,7 +12,7 @@ export default function Reviews() {
     const params = useParams();
     let paramsObject = {}; 
 
-    if (params.slug) { paramsObject.category = params.slug };
+    if (params.category) { paramsObject.category = params.category };
```

Comment:
I am sorry if my feedback on this hasn't been very clear, but this code is redundant. All you are doing is creating a new Object and assigning params.category to a key that is named the same on the new object, when you could be using params in the first place.